source/overview.rst

```diff
@@ -80,10 +88,19 @@ The WEC-Sim input file (``wecSimInputFile.m``) for the OSWEC initializes and spe
    :language: matlab
         
 
+.. note::
+    I would probably say that WEC-Sim is object oriented before you start 
+    talking about classes. Maybe a link to `help 
+    <https://uk.mathworks.com/discovery/object-oriented-programming.html>`_ 
+    would be useful for the uninitiated.
+
```

Comment:
@H0R5E that's a good suggestion, can you submit this revision via a PR?

Reply:
#456